### ISSUE TITLE
There have  AwesomeTTS with AWS Polly support version on github. could you merge it to yours?  thank  you

### ISSUE THREAD

**Issue by HeroBlackInk** [kelciour/awesometts-anki-addon#2]
here is link.

https://github.com/artem7902/awesometts-anki-addon

**Comment by kelciour** [comment]
I'm sorry, but I'm not interested in spending my time on supporting it. Though, I can make something similar by commission in a week or two, that would be maybe a bit easier to install and won't require any extra dependencies. I may make something on my own but it won't happen any time soon as I have a couple of unfinished projects and a lot of Anki decks that I'd like to do first.

The above mentioned repository worked fine for me the last time I provided paid help to install it. Though, one file is still needed to be updated to make it work. I looked up my messages from that time and here's instructions that might make it a bit easier to install:

1. Download the add-on https://github.com/artem7902/awesometts-anki-addon by using green `Clone or download` button and selecting `Download ZIP`.

2. Open the add-ons folder via `Tools - Add-ons - Open Add-ons Folder` and unpack it there, i.e. after unpacking it'll be something like `C:\Users\Nickolay\AppData\Roaming\Anki2\addons21\awesometts-anki-addon-master`

3. Use `Download ZIP` button to download from <a href="https://gist.github.com/kelciour/ccf422a69f92f190b6402d730b7285a0" target="_blank">https://gist.github.com/kelciour/ccf422a69f92f190b6402d730b7285a0</a> and export `__init__.py` into the `awesometts-anki-addon-master` folder, i.e. replace `__init__.py` in `awesometts-anki-addon-master` with `__init__py` from the archive.

4. Create Amazon AWS account. It will require a credit card - https://aws.amazon.com/polly/pricing/?nc1=h_ls

5. After registration, get authentication credentials via IAM Console - https://boto3.amazonaws.com/v1/documentation/api/latest/guide/quickstart.html#configuration

    In IAM Console click on Users to create a new user, then Add user, type user name and check "Programmatic access", then click on Next:Permissions, then Attach Existing policies directly, in search box print Polly and check AmazonPollyFullAccess, AmazonPollyReadOnly, then Next:Review, Create use and copy Access key and Secret Key or leave the page opened.

6. Close Anki, download the attached archive [aws.zip](https://github.com/kelciour/awesometts-anki-addon/files/2947087/aws.zip) and unpack it into your home folder (`C:\Users\Nickolay`), i.e. `C:\Users\Nickolay\.aws`, and add your keys (`aws_access_key_id` and `aws_secret_access_key`) into `.aws\credentials` file. It can be opened in any text editor.

That's it. Maybe some steps can be simplified a bit.

**Comment by HeroBlackInk** [comment]
you are so kind.

**Comment by kelciour** [comment]
Ah, no problem. I'll leave it open just in case if someone decided to ask the same question or I may happen to implement it myself.

**Comment by lucwastiaux** [comment]
This looks like something we should merge. Just curious, what is the advantage of AWS polly ? we've merged Google Cloud TTS. I'm working on Azure TTS which has some neural voices as well. It would make sense to merge Amazon Polly in case it provides some unique voice combination which is not found elsewhere.

Separately, I found a cool way to test the various services automatically: download a voice file, then run it through a speech recognition service ( I use Azure cognitive services). I have a change which i'll integrate soon. This should allow us to detect when services start breaking.
